src/components/CommentsList.jsx

```diff
@@ -1,5 +1,31 @@
+import { useState, useEffect } from "react";
+import { getAllComments } from "../utils.js";
+import CommentCard from './CommentCard.jsx'
+
 function CommentsList() {
-    return <h1>CommentsList</h1>
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+
+useEffect(() => {
+    getAllComments(1).then((result) => {
+        setComments(result)
+        setIsLoading(false)
```

Comment:
we also want to set loading to false in the catch block so that if the request fails, the component doesn't just display "loading" the whole time